src/runner.rs

```diff
@@ -249,20 +305,56 @@ impl TestConfig {
 
         let total_tests = outputs.len();
         let mut failing_tests = 0;
+        let mut can_be_fixed_with_overwrite_tests = 0;
+        let mut updated_tests = 0;
         for result in &outputs {
-            if let Err(error) = result {
-                eprintln!("{}", error);
-                failing_tests += 1;
+            match result {
+                Ok(_) => {}
+                Err(error) => {
+                    match error {
+                        InnerTestError::TestUpdated { .. } => {
+                            updated_tests += 1;
+                        }
+
+                        InnerTestError::TestFailed { .. } => {
+                            can_be_fixed_with_overwrite_tests += 1;
+                            failing_tests += 1;
+                        }
+
+                        InnerTestError::IoError(_, _)
+                        | InnerTestError::CommandError(_, _, _)
+                        | InnerTestError::ErrorParsingExitStatus(_, _, _)
+                        | InnerTestError::ErrorParsingArgs(_, _) => {
+                            failing_tests += 1;
+                        }
+                    }
+                    eprintln!("{}", error);
+                }
             }
         }
 
-        println!(
-            "ran {} {} tests with {} and {}\n",
-            total_tests,
-            "golden".bright_yellow(),
-            format!("{} passing", total_tests - failing_tests).green(),
-            format!("{} failing", failing_tests).red(),
-        );
+        if self.overwrite_tests {
+            println!(
+                "ran {} {} tests with {} and {}\n",
+                total_tests,
+                "golden".bright_yellow(),
+                format!("{} passing", total_tests - failing_tests).green(),
+                format!("{} failing", failing_tests).red(),
+            );
+        } else {
+            println!(
+                "ran {} {} tests with {}, {} and {}\n",
+                total_tests,
+                "golden".bright_yellow(),
+                format!("{} passing", total_tests - failing_tests).green(),
+                format!("{} failing", failing_tests).red(),
+                format!("{} updated", updated_tests).cyan(),
```

Comment:
Is this backwards? We print the updated tests to the output if `!self.overwrite_tests`

Reply:
Yes, this was backwards. I just found it myself. xD